packages/react-dom/src/shared/trustedTypesAwareToString.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+/**
+ * Returns true only if Trusted Types are available in global object and the value
+ * is a trusted type.
+ */
+function isTrustedTypesValue(value: any) {
+  if (typeof window.TrustedTypes === 'undefined') {
```

Comment:
Is there any reason for the `window.` qualifier here? Can we write plain `typeof TrustedTypes === 'undefined'`?

Generally in `react-dom` source we try to avoid assuming `window` exists because (depending on the code path) some `react-dom` logic may execute even on the server environment. Such as when some module does `require('react-dom')`. So we want to be careful to not introduce accidental reference errors there.